test/ctia/entity/asset_mapping_test.clj

```diff
@@ -73,12 +75,11 @@
                                          "foogroup"
                                          "user")
 
-     (let [ids (helpers/POST-entity-bulk
-                app
-                new-asset-mapping-maximal
-                :asset_mappings
-                30
-                {"Authorization" "45c1f5e3f05d0"})]
+     (let [ids (-> app
+                (helpers/POST-bulk
+                 (-> (fixtures/bundle 10 true)
+                     (select-keys [:assets :asset_mappings])))
+                :asset_mappings)]
```

Comment:
Ah, missed that one. BTW, the other day I discussed with Victor the possibility of moving IROH's `user.format/clean-staged-namespaces!` to CTIA. Turns out it's not as simple. According to him: 

> (This) isn't so trivial, because formatting-stack and user.* are densely/delicately configured

Maybe if I annoy him with inconsistent formatting and indentation he'd finally extract and bring it over here :)